src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
Sounds like a different PR to this one. I think I'll keep it as it is, so there isn't any scope creep in this PR.